composer.json

```diff
@@ -13,11 +13,11 @@
         }
     ],
     "require": {
-        "php": ">=5.3.0",
-        "guzzle/guzzle": ">=3.6"
+        "php": ">=5.5.0",
+        "guzzlehttp/guzzle": "^6.2.0"
     },
     "require-dev": {
-       "phpunit/phpunit": ">=4.0"
+       "phpunit/phpunit": "^4.0.0"
```

Comment:
composer prefers the caret. Also `^4.0.0` is equivalent to `>= 4.0.0 && < 5.0.0`